emission/tests/storageTests/TestTokenQueries.py

```diff
@@ -73,21 +76,73 @@ def test_auth_fail_file_insert(self):
         tlm = enat.TokenListMethod()
         self.assertEqual(tlm.verifyUserToken('b'), 'b')
 
-    def test_run_script_auth_fail(self):
-        os.system("python3 bin/auth/insert_tokens.py emission/tests/storageTests/token_test_files/tokens.txt")
+    def test_run_script_file_auth_fail(self):
+        os.system("python3 bin/auth/insert_tokens.py --file emission/tests/storageTests/token_test_files/tokens.txt")
         with self.assertRaises(ValueError):
             tlm = enat.TokenListMethod()
             tlm.verifyUserToken('z')
 
+    def test_run_script_single(self):
+        os.system("python3 bin/auth/insert_tokens.py --single a")
+        os.system("python3 bin/auth/insert_tokens.py --single b")
+        self.assertEqual(esdt.get_all_tokens(), ['a','b'])
+
+    def test_run_script_single_auth(self):
+        os.system("python3 bin/auth/insert_tokens.py --single a")
+        os.system("python3 bin/auth/insert_tokens.py --single b")
+        tlm = enat.TokenListMethod()
+        self.assertEqual(tlm.verifyUserToken('b'), 'b')
+
+    def test_run_script_single_auth_fail(self):
+        os.system("python3 bin/auth/insert_tokens.py --single a")
+        os.system("python3 bin/auth/insert_tokens.py --single b")
+        with self.assertRaises(ValueError):
+            tlm = enat.TokenListMethod()
+            tlm.verifyUserToken('z')
+
+    def test_run_script_uuid(self):
+        edb.get_uuid_db().insert_one({"user_email":"a"})
+        edb.get_uuid_db().insert_one({"user_email":"b"})
+        edb.get_uuid_db().insert_one({"user_email":"c"})
+        edb.get_uuid_db().insert_one({"user_email":"d"})
+        os.system("python3 bin/auth/insert_tokens.py --uuid")
+        self.assertEqual(esdt.get_all_tokens(), ['a','b','c','d'])
+
+    def test_run_script_uuid_auth(self):
+        edb.get_uuid_db().insert_one({"user_email":"a"})
+        edb.get_uuid_db().insert_one({"user_email":"b"})
+        edb.get_uuid_db().insert_one({"user_email":"c"})
+        edb.get_uuid_db().insert_one({"user_email":"d"})
+        os.system("python3 bin/auth/insert_tokens.py --uuid")
+        tlm = enat.TokenListMethod()
+        self.assertEqual(tlm.verifyUserToken('b'), 'b')
+
+    def test_run_script_uuid_auth_fail(self):
+        edb.get_uuid_db().insert_one({"user_email":"a"})
+        edb.get_uuid_db().insert_one({"user_email":"b"})
+        edb.get_uuid_db().insert_one({"user_email":"c"})
+        edb.get_uuid_db().insert_one({"user_email":"d"})
+        os.system("python3 bin/auth/insert_tokens.py --uuid")
+        with self.assertRaises(ValueError):
+            tlm = enat.TokenListMethod()
+            tlm.verifyUserToken('z')
+
+    def test_run_script_show(self):
+        esdt.insert({'token':'x'})
+        esdt.insert({'token':'y'})
+        esdt.insert({'token':'z'})
+        os.system("python3 bin/auth/insert_tokens.py --show")
+        self.assertEqual(True, True)
```

Comment:
Nit: you should be able to read the script result by opening the stdout of the process as a file and reading from it.
https://docs.python.org/3/library/subprocess.html#subprocess.run

This may be too overkill for the current code, but I would encourage you to figure it out as a learning experience in your spare time.